n2y/export.py

```diff
@@ -58,6 +59,23 @@ def export_page(
 ):
     page_properties = _page_properties(page, pandoc_format, id_property, url_property, property_map)
     pandoc_ast = page.to_pandoc()
+    has_tables = pandoc_ast and any(isinstance(e, Table) for e in pandoc_ast[1])
+
+    if has_tables:
```

Comment:
Perhaps you could split this out into a separate function, say "_num_headerless_tables"?